services/web/server/tests/unit/with_dbs/01/test_api_keys_rpc.py

```diff
@@ -95,21 +101,21 @@ async def rpc_client(
     return await rabbitmq_rpc_client("client")
 
 
-async def test_api_key_get(
-    fake_user_api_keys: list[str],
+async def test_get_api_key(
+    fake_user_api_keys: list[ApiKey],
     rpc_client: RabbitMQRPCClient,
     osparc_product_name: ProductName,
     logged_user: UserInfoDict,
 ):
-    for api_key_name in fake_user_api_keys:
+    for api_key in fake_user_api_keys:
```

Comment:
Since the RPC endpoints are usually called within our codebase. We have a convention that allows for reuse of code and ensures endpoint callas are always tested. Please refactor the RPC endpoint's test to use the following pattern, that we use for all other endpoints.

It works like this:
1. define a function to call the RPC endpoint inside the service_library
2. add the test in this module to it
3. everybody calling that shared function can profit form it's code being covered and avoids introducing code duplication and issue related to it.


You can see an example if you see how this endpoint is used. https://github.com/ITISFoundation/osparc-simcore/blob/6eb9b2efefad6259cf1e88e812bce311694fc37a/packages/service-library/src/servicelib/rabbitmq/rpc_interfaces/dynamic_scheduler/services.py#L51

Reply:
I would suggest to refactor all the code in this module with this pattern.